src/_selectors/FullTradeSelectors.js

```diff
@@ -255,15 +255,24 @@ const availableAssetsFilter = (assets, times, now) => {
         if (!s.times) {
             return;
         }
-        const open = s.times.open[0];
-        const close = s.times.close[0];
+        const open = s.times.open;
+        const close = s.times.close;
 
         // Assuming closing time is larger than open time
         // TODO: some of Random does have time which does not fit into assumption
         if (s.name.indexOf('Random') > -1) {
             availabilities[s.symbol] = true;
-        } else if (timeStringBigger(nowInTimeString, open) && timeStringSmaller(nowInTimeString, close)) {
-            availabilities[s.symbol] = true;
+        } else if (open.length === 1) {
+            if (timeStringBigger(nowInTimeString, open[0]) && timeStringSmaller(nowInTimeString, close[0])) {
+                availabilities[s.symbol] = true;
+            }
+        } else if (open.length === 2) {
+            if (
+                (timeStringBigger(nowInTimeString, open[0]) && timeStringSmaller(nowInTimeString, close[0])) ||
```

Comment:
Definitely we will have a transform on the server_data inputs. We can even have it as an extension to live-api. Not today, but lets consider it soon.